sigchanyzer.go

```diff
@@ -48,17 +48,25 @@ func run(pass *analysis.Pass) (interface{}, error) {
 			return
 		}
 		var chanDecl *ast.CallExpr
+
+		// track whether call.Args[0] is a *ast.CallExpr. This is true when make(chan os.Signal) is passed directly to signal.Notify
+		wasCallExpr := false
+
 		switch arg := call.Args[0].(type) {
 		case *ast.Ident:
 			if decl, ok := findDecl(arg).(*ast.CallExpr); ok {
 				chanDecl = decl
+				wasCallExpr = false
 			}
 		case *ast.CallExpr:
```

Comment:
We can just skip this case entirely, because `chanDecl` will be nil and the check below will be fired up. I think the main point left is whether we should skip this entirely, or just for `make` builtin?

Reply:
@cuonglm didn't skip this and added a check to make sure the call is a make builtin i.e. ```make(chan os.Signal)``` or ```make(chan os.Signal, 1)```